src/test/java/PokeLifeBotByMetin/pages/LogInPage.java

```diff
@@ -0,0 +1,31 @@
+package PokeLifeBotByMetin.pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class LogInPage {
+
+    @FindBy(name = "login")
+    private WebElement loginInput;
+
+    @FindBy(name = "haslo")
+    private WebElement passInput;
+
+    @FindBy(xpath = "//button[@class='btn btn-primary']")
+    private WebElement logInButton;
+
+    private WebDriver driver;
+
+    public LogInPage(WebDriver driver) {
+        PageFactory.initElements(driver, this);
+        this.driver = driver;
+    }
+
+    public void login(String username, String password) {
+        loginInput.sendKeys(username);
+        passInput.sendKeys(password);
+        logInButton.click();
+    }
+}
```

Comment:
One line blank keeps us face from deleting curly braces. Add it in everywhere :)

Reply:
I don't understand :(

Reply:
Add an empty line at the end (in this case line 32), and repeat in each file.